### PR TITLE
LLT-5246 Add netsh checks on config changes in win VM

### PR DESCRIPTION
**Pull request by matislovas**
### Problem
Some commands in VM do not generate results instantly. This was observed primarily on Windows, when setting IP addresses on interface using `netsh` command.

### Solution
After command execution, launch a "check-and-grep-like" commands for some predetermined amount of time, to search for a command execution result.

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

File: nat-lab/tests/utils/command_grepper.py
Comment (mathiaspeters) [inline nat-lab/tests/utils/command_grepper.py:40]: Nit: `exp` should be renamed to `exp_primary` for consistency reasons and to make the "primary" reference in the doc comment clearer

File: nat-lab/tests/utils/command_grepper.py
Comment (mathiaspeters) [inline nat-lab/tests/utils/command_grepper.py:54]: Is it expected to rerun the command on each iteration?

Comment (matislovas) [reply nat-lab/tests/utils/command_grepper.py:54]: Yup, that's the idea. Run _check_ command every second, and when it succeeds - `return True`. Or timeout and die.

File: nat-lab/tests/utils/command_grepper.py
Comment (matislovas) [reply nat-lab/tests/utils/command_grepper.py]: Fixed

**Comment by mathiaspeters** [comment]
+1

**Comment by ThrasherLT** [comment]
lgtm +10 maybe would like it if there were less duplicated code between the exists and not_exists functions, but their usage seems convenient, so won't block the merge on that

**Review by ThrasherLT — APPROVED** [review]

**Pull request merged**
